### PR TITLE
Update PHP CS Fixer config to v2.18

### PR DESCRIPTION
**Pull request by keradus**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -11,8 +11,6 @@ return PhpCsFixer\Config::create()
         '@Symfony' => true,
         '@Symfony:risky' => true,
         'protected_to_private' => false,
-        'native_constant_invocation' => true,
```
Comment (keradus) [inline .php_cs.dist:14]: in continuation of https://github.com/symfony/symfony/pull/39772/files#r554500252 ,

@nicolas-grekas

would you like to update this rule too?
https://cs.symfony.com/doc/rules/function_notation/native_function_invocation.html#strict

**Review by nicolas-grekas — APPROVED** [review]

**Comment by fabpot** [comment]
FYI, fabbot is using 2.18 since yesterday.

**Comment by keradus** [comment]
awesome, @fabpot !

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you @keradus.

**Pull request merged**